reproman/support/jobs/orchestrators.py

```diff
@@ -486,7 +486,8 @@ def __init__(self, resource, submission_type, job_spec=None,
                  resurrection=False):
         if not external_versions["datalad"]:
             raise MissingExternalDependency(
```

Comment:
Just a note: the datalad-bump branch already switched this over to `external_versions.check()`, which results results in `.name` being set in the `MissingExternalDependency` instance.

Reply:
Oh cool, One more reason to have that pr merged, which I guess I will do with adding datalad rc into some dependency within setup.py .